content/docs/migration_1.0/C++.md

```diff
@@ -0,0 +1,363 @@
+---
+title: "C++"
+weight : 6300
+menu:
+  docs:
+    parent: migration_1.0
+---
+
+
+Zenoh 1.0.0 brings a number of changes to the API, with a concentrated effort to bring the C++ API to more closely resemble the Rust API in usage.
+
+The improvements we bring in this update include:
+
+- A simpler organization of the Zenoh classes, abstracting away the notion of View and Closure.
+- Improved and more flexible Error Handling through error codes and exceptions.
+- Support for serialization of common types like strings, numbers, vectors through Codecs.
+- Ability for users to define their own Codecs (for their own types or to overwrite the default one)!
+- Improved stream handlers and callback support.
+- Simpler attachment API.
+
+Now that the *amuse bouche* is served,  let's get into the main course!
+
+## Error Handling
+
+Prior to 1.0.0, all Zenoh call failures were handled by either returning a `bool` value indicating success or failure (and probably returning a error code) or returning an `std::variant<ReturnType, ErrorMessage>`. For instance:
```

Comment:
In other parts of the migration guide we explicitly mention `0.11.0`. Let's make sure we clearly state `0.11.0` as a previous version. E.g. `0.7.X` had yet again a different API.

Reply:
Updated in recent commit